### DISCUSSION TITLE
Please terach me your favorite McDonald's Menu

### DISCUSSION THREAD

**Discussion by paaaaay5** [NAIST-SE/AttractingContributors#482] [General]
This is my repository url.(https://github.com/paaaaay5/Favorite-McDonalds-Menu)
Please access and write your favorite menu. Thank you.

**Comment by XC1996114514** [comment]
I like the Chicken McNuggets

So could you help me to filled the readme?
Here is my repository;
https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam

Thanks a lot for helping me!
